src/actions/AccountReferralActions.ts

```diff
@@ -258,13 +259,25 @@ export async function validatePromoCardsInner(dataStore: EdgeDataStore, cards: M
       if (card.maxBuildNum < buildNum) continue
     }
 
+    const version = funcs.getVersion()
+    if (typeof card.version === 'string') {
+      if (card.version !== version) continue
+    }
```

Comment:
Shouldn't `fungs.getVersion` be called inside the `if` statement? Otherwise, we don't actually care what the version is.

Reply:
yep. that's fair. will change